h/static/scripts/controllers/lozenge-controller.js

```diff
@@ -1,70 +1,65 @@
 'use strict';
 
-const escapeHtml = require('escape-html');
-
 const Controller = require('../base/controller');
 const searchTextParser = require('../util/search-text-parser');
+const { setElementState } = require('../util/dom');
 
 /**
- * Create a lozenge with options.content as its content and append it to containerEl.
+ * A lozenge representing a single search term.
  *
- * A lozenge is made of two parts - the lozenge content and
- * the 'x' button which when clicked removes the lozenge
- * from the container and executes the delete callback provided.
+ * A lozenge consists of two parts - the lozenge content and the 'x' button
+ * which when clicked calls the `deleteCallback` handler passed in the
+ * controller's options.
  *
- * var lozenge = new Lozenge(containerEl, {
- *   content: content,
- *   deleteCallback: deleteCallback,
+ * const lozenge = new Lozenge(element, {
+ *   content,
+ *   deleteCallback,
  * });
  */
 class LozengeController extends Controller {
-  constructor(containerEl, options) {
```

Comment:
This controller used to take the container of the lozenges as input, which was weird because none of the controllers work like this. Now it works like other controllers, taking either a server-rendered lozenge or one cloned from a template on the client as input.